homeassistant/components/conversation/__init__.py

```diff
@@ -297,6 +298,65 @@ async def websocket_list_agents(
     connection.send_message(websocket_api.result_message(msg["id"], {"agents": agents}))
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "conversation/agent/homeassistant/debug",
+        vol.Required("sentence"): vol.All(cv.ensure_list, [cv.string]),
```

Comment:
let's not use `ensure_list` and just always expect a list. Also, it's not coming from YAML so just expect a string.

```suggestion
        vol.Required("sentence"): [str],
```

Reply:
Also changed input name to "sentences" since it will always be a list.